subgraphs/loreum-nft/abis/LoreumNFT.json

```diff
@@ -521,7 +558,7 @@
         "type": "string"
       }
     ],
-    "stateMutability": "view",
```

Comment:
In our contract, `tokenURI(uint256)` - the function that returns the full URI - was updated to `public` because we updated the modifier on the string to remove the `constant` value, thus making it `view` and not `pure`

`function tokenURI(uint256 tokenId_) override view public returns (string memory)`

Reply:
need to use npm for to get the ABIs